hal/src/stm32f2xx/core_hal_stm32f2xx.c

```diff
@@ -240,6 +240,11 @@ void HAL_Core_Config(void)
 #ifdef HAS_SERIAL_FLASH
     //Initialize Serial Flash
     sFLASH_Init();
+#if PLATFORM_ID == PLATFORM_DUO_PRODUCTION
```

Comment:
I suggest factoring out the target flash device to it's own #define in one place then referencing that each time, rather than repeating the compile time conditional each time it's used.